src/main/java/com/extendedclip/deluxemenus/DeluxeMenus.java

```diff
@@ -102,6 +102,20 @@ public void onEnable() {
 
     this.adventure = BukkitAudiences.create(this);
 
+    setupItemHooks();
+
+    if (Bukkit.getPluginManager().getPlugin("Vault") != null) {

```

Comment:
I think `isPluginEnabled()` is better to check if a plugin is installed **and** enabled.

Reply:
Done